File: 1637-widest-vertical-area-between-two-points-containing-no-points/1637-widest-vertical-area-between-two-points-containing-no-points.cpp
class Solution {
public:
    int maxWidthOfVerticalArea(vector<vector<int>>& points) {
        sort(points.begin(), points.end());
        int wide = 0;
        for (int i = 0; i < points.size() - 1; i++) {
            if (points[i+1][0] - points[i][0] > wide) {
                wide = points[i+1][0] - points[i][0];
            }
        }
        return wide;
    }
};

File: LeetCode/1637-widest-vertical-area-between-two-points-containing-no-points/1637-widest-vertical-area-between-two-points-containing-no-points.py
class Solution:
    def maxWidthOfVerticalArea(self, points: List[List[int]]) -> int:
        points.sort()
        wide = 0
        for i in range(len(points) - 1):
            if points[i+1][0] - points[i][0] > wide:
                wide = points[i+1][0] - points[i][0]
        return wide